lib/core/src/lib/form/components/widgets/date/date.widget.html

```diff
@@ -1,6 +1,6 @@
 <div class="{{field.className}}" id="data-widget" [class.adf-invalid]="!field.isValid && isTouched()">
     <mat-form-field class="adf-date-widget" [hideRequiredMarker]="true">
-        <label class="adf-label" [attr.for]="field.id">{{field.name | translate }} ({{field.dateDisplayFormat}})<span class="adf-asterisk"
+        <label class="adf-label" [attr.for]="field.id">{{field.name | translate }} ({{dateFormatTranslationService.convertDateFnsToMomentFormat(field.dateDisplayFormat)}})<span class="adf-asterisk"
```

Comment:
please extract to a separate function so that template does not deal with services conversion directly:

```ts
formatLabel(field: FormField): string {
  const displayName = this.translationService.instant(field.name);
  const displayFormat = this.dateFormatTranslationService.convertDateFnsToMomentFormat(field.dateDisplayFormat);

  return `${displayName} (${displayFormat})`; 
}
```

then in HTML:

```html
<label>{formatLabel(field)}</label>
```

also provide unit tests for the formatLabel function as it becomes easier

Reply:
implemented